app/javascript/packages/phone-input/index.ts

```diff
@@ -225,6 +219,12 @@ export class PhoneInputElement extends HTMLElement {
     }
   }
 
+  getInvalidFormatMessage(countryCode: CountryCode): string {
+    return countryCode === 'US'
+      ? this.strings.invalid_phone_us || ''
```

Comment:
Another approach to writing this function, and simplifying the string specifications, might be the following:

```
getInvalidFormatMessage(countryCode: CountryCode): string {
    return countryCode === 'US'
      ? t('errors.messages.invalid_phone_number.us')
      : t('errors.messages.invalid_phone_number.international');
}
```

Along with adding `import { t } from '@18f/identity-i18n';` in the same file.

Glancing at the code quickly, I think that would make it simpler. Maybe I'm overlooking a dependency elsewhere.